src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/navi_set_video_config_request.cc

```diff
@@ -143,8 +141,8 @@ void NaviSetVideoConfigRequest::onTimeOut() {
   }
 
   std::vector<std::string> empty;
-  application_manager_.OnStreamingConfigured(
-      app->app_id(), protocol_handler::ServiceType::kMobileNav, false, empty);
+  application_manager_.OnStreamingConfigurationFailed(
+      app->app_id(), empty, std::string());
```

Comment:
Shouldn't a reason message be provided in this case as well?

Reply:
Reason is added in 1c6840d4d5edcef26befb640c2dcc57a9c2fa20c